pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+##### Description:
```

Comment:
Personally, I would prefer the headers to be a little bigger, it looks smaller than the text that it includes.